client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-content/motion-content.component.html

```diff
@@ -105,6 +105,9 @@
 
     <!-- The HTML Editor for motions and traditional amendments -->
     <ng-container *ngIf="editMotion && !isParagraphBasedAmendment">
+        <h3>
```

Comment:
You might want to check if a preamble is set. 

```suggestion
        <h3 *ngIf="showPreamble && preamble">
```